vcf-dumper/README.md

```diff
@@ -0,0 +1,122 @@
+
+Once we got a server with data, and a jar or war artifact pointing to it, we can try some queries.
+
+### Queries
+The VCF dumper has two main interfaces that can be used to execute queries over the archive and get the result in VCF format:
+* **WebServices**: allows to dump the variants of some study(ies) that are in a given genomic region (or list of regions). Queries with no region are not allowed
+* **Command Line Interface**: for EVA internal use only. Allows to dump all the variants for a given study(ies) and file(s). It does not include region filters
+
+#### Webservices
+The Webservices API contains actually a single endpoint: `{baseURL}/{regionId}/variants`.
+{regionId} can be a single region or a comma separated list of regions. Each region is a composed of a chromosome name, and optionally a ':' character followed by natural numbers pair (start and end), separated by '-'. Some examples of valid regions are:
+* `1`
+* `chr1`
+* `MT`
+* `1:1000-2000`
+* `chr1:1000-2000`
+
+And an example of a region list (whole chromosome 1, chromosome 2 positions 1000 to 2000, chromosome X positions 4000 to 5000) :
+* `1,2:1000-2000,X:4000-5000`
+
+The variants endpoint also include some URL variables. Two are mandatory:
+* **species**: the queried database name will be the species value prefixed by "eva_" (e.g., for species *hsapiens*, this service will fetch the data from a database named *eva_hsapiens*)
+* **studies**: comma separated list of studies ids to query
+
+In addition to those, there are several optional URL variables:
+* annot-ct
+* maf
+* polyphen
+* sift
+* ref
+* alt
+* miss_alleles
+* miss_gts
+
+Any time a valid call is invoked, the client will receive a VCF file stream, containing the variants that satisfy the query criteria. 
+#### CLI
+The Command Line Interface is intended to dump whole studies in VCF format. It does not allow filtering by region, so all the variants in the study will be dumped. This may be a time consuming operation, especially for big studies. For that reason, this tool is thought to be used only internally by EVA.
+
+This command line interface includes the following parameters:
+* **species**: species the data are associated with
+* **database**: name of the database to extract data from
+* **outdir**: output directory
+* **studies**: comma separated list of studies to query
+* **files**: comma separated list of files to query (each study in EVA can be composed by one or many files, as described [here](https://github.com/EBIvariation/eva-pipeline/wiki/MongoDB-schema#files)
+
+A successfull command execution will produce a VCF file in the output directory.
+
+#### Querying the test data
+For querying the test databases we need some values for the mandatory params:
```

Comment:
To query the test databases, values must be assigned to the following, mandatory parameters